.github/workflows/openshift-ci/Containerfile

```diff
@@ -0,0 +1,65 @@
+# Use the fedora image as the base image
```

Comment:
Why do you have 3 Containerfiles variants?

Reply:
cruft, I need top remove the other ones